lib/shared/messages.ts

```diff
@@ -38,129 +38,116 @@ export const info = {
 	) => {
 		const targets = [];
 		if (failed + successful === 1) {
-			targets.push(`to ${drive.description} (${drive.displayName})`);
+			targets.push(
+				i18next.t('message.to') + `${drive.description} (${drive.displayName})`,
+			);
 		} else {
 			if (successful) {
-				const plural = successful === 1 ? '' : 's';
-				targets.push(`to ${successful} target${plural}`);
+				targets.push(
+					i18next.t('message.to') +
```

Comment:
As mentioned earlier, IMHO it's not safe to simply concatenate strings together when dealing with translation, as different languages might need different parts of the string to be displayed in a different order.

Reply:
I sincerely thank you for your professional guidance!